index.js

```diff
@@ -138,6 +139,37 @@ module.exports = {
     // https://reactjs.org/docs/hooks-rules.html#eslint-plugin
     'react-hooks/rules-of-hooks': 'error',
 
+    // Alphabetical sorting is the most reliable way to make code consistently readable
```

Comment:
[is it the most useful](https://github.com/americanexpress/eslint-config-amex/pull/20#issuecomment-621302651)? If not, what are the tradeoffs and why was reliability chosen over utility? (please add the answers, etc to the comment for future maintainers)